docs/repositories-configure/adjusting-quality-gates.md

```diff
@@ -33,31 +23,9 @@ The following screenshot displays the default configuration values:
 
         To ensure that the changes in each pull request have a minimum level of coverage, use the gate **Diff coverage is under** instead.
 
-!!! note
-    Learn how Codacy calculates the code quality metrics in more detail:
-
-    -   [Which code quality metrics does Codacy calculate?](../faq/code-analysis/which-metrics-does-codacy-calculate.md)
-    -   [Why does Codacy show unexpected coverage changes?](../faq/code-analysis/why-does-codacy-show-unexpected-coverage-changes.md)
-
```

Comment:
Let's not remove this part, it's important to help give more context to users about how exactly does Codacy calculate each of the metrics. The user configuring the gates or goals needs to understand how the metrics work so they configure appropriate rules.

It's also better to repeat this both for Gates and Goals - so we should create a Markdown file under `docs/assets/includes/` and then reuse it on both pages.